src/components/MediatedRequestsActivities/components/MediatedRequestsFilters/MediatedRequestsFilters.test.js

```diff
@@ -39,15 +39,15 @@ describe('MediatedRequestsFilters', () => {
     expect(screen.getByTestId(testIds.mediatedRequestsFiltersSearch)).toBeInTheDocument();
   });
 
-  it('should trigger search with correct props', () => {
+  it('should render search with correct props', () => {
     expect(Search).toHaveBeenCalledWith(expect.objectContaining({}), {});
   });
 
   it('should render status filter', () => {
     expect(screen.getByTestId(testIds.mediatedRequestFiltersStatusFilter)).toBeInTheDocument();
   });
 
-  it('should trigger status filter with correct props', () => {
+  it('should render status filter with correct props', () => {
```

Comment:
I think trigger was better because in the test you check component call, not component render.

Reply:
Fixed